pkg/cmd/admin/network/project_options.go

```diff
@@ -84,7 +84,14 @@ func (p *ProjectOptions) Validate() error {
 		errList = append(errList, errors.New("must provide --selector=<project_selector> or projects"))
 	}
 
-	// TODO: Validate if the openshift master is running with mutitenant network plugin
+	clusterNetwork, err := p.Oclient.ClusterNetwork().Get(sdnapi.ClusterNetworkDefault)
+	if err != nil {
+		errList = append(errList, errors.New("could not manage pod network: error on getting the running network plugin"))
```

Comment:
nit: 'could not manage pod network' is redundant, just throw the error msg something like 'Failed to fetch current network plugin info'